DomainModels/Helpers/CustomerProfile.cs

```diff
@@ -0,0 +1,14 @@
+﻿using AutoMapper;
+using DomainModels;
+using DomainServices.Dtos;
+
+namespace DomainServices.Profiles
+{
+    public class CustomerProfile : Profile
+    {
+        public CustomerProfile()
+        {
+            CreateMap<Customer, CustomerDto>().ReverseMap();
```

Comment:
Não é uma boa prática fazer uso do ReverseMap.
```suggestion
            CreateMap<Customer, CustomerDto>();
```